fulp_modules/jobs/antag_protection_overwrites.dm

```diff
@@ -1,62 +1,170 @@
 /// Secret mode
 /datum/game_mode/traitor/New()
-	protected_jobs += list("Brig Physician", "Deputy")
+	protected_jobs += list(
+	"Brig Physician",
+	"Deputy",
+	"Head of Personnel",
+	"Research Director",
+	"Chief Engineer",
+	"Chief Medical Officer",
+	)
+	. = ..()
+
+/// We'll allow Command antags for some gamemodes to help prevent metagaming
+/datum/game_mode/traitor/internal_affairs/New()
+	protected_jobs -= list(
+	"Head of Personnel",
+	"Research Director",
+	"Chief Engineer",
+	"Chief Medical Officer",
+	)
 	. = ..()
 
 /datum/game_mode/cult/New()
-	protected_jobs += list("Brig Physician", "Deputy")
+	protected_jobs += list(
+	"Brig Physician",
+	"Deputy",
+	)
 	. = ..()
 
 /datum/game_mode/revolution/New()
-	protected_jobs += list("Brig Physician", "Deputy")
+	protected_jobs += list(
```

Comment:
Why hello useless code.
Heads of Staff are already protected from becoming revolutionaries.